migrations/Version202205232110572348_generis.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\generis\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\generis\model\data\Ontology;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use core_kernel_persistence_smoothsql_SmoothModel as SmoothModel;
+
+final class Version202205232110572348_generis extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Fix ontology models.';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $serviceLocator = $this->getServiceLocator();
+
+        /** @var SmoothModel $ontology */
+        $ontology = $this->getServiceLocator()->get(Ontology::SERVICE_ID);
+
+        $this->fixModels($ontology, SmoothModel::OPTION_READABLE_MODELS);
+        $this->fixModels($ontology, SmoothModel::OPTION_WRITEABLE_MODELS);
+        $this->fixModel($ontology, SmoothModel::OPTION_NEW_TRIPLE_MODEL);
+
```

Comment:
Also please remove the `SmoothModel::OPTION_CACHE_SERVICE` part like here: https://github.com/oat-sa/generis/commit/061126ad186952f759d309ec0d1731f6ae8266d0#diff-801705e555d09d2a1a4cfb1454765ff1570f123b566efb15ccff7e380232da04L16

Reply:
The problem was that `config/generis/ontology.conf.php` was not updated along with code changes, as was discussed in the related ticket, so the scope is to make this config file up to date, including the removal of this `cache` part, investigating that it won't break anything also in same scope 🙂